### ISSUE TITLE
Better hunting

### ISSUE THREAD

**Issue by doomgrave** [AnthonyOSX/GO-Bot#1]
Instead of finding pokemon nearby pokestop, it could not use the script used for the map to obtain more rare pokemon exact coordinates?

**Comment by doomgrave** [comment]
It is an improving idea, not an issue :)

**Comment by AnthonyOSX** [comment]
Ideally yes, however teleporting to only "rare" Pokemon locations could raise a red flag to Niantic that you may be cheating. Further development has to be done first :)

**Comment by MarcusOy** [comment]
Agreed. I played Ingress, which was Niatic's previous game, and i occasionally used a GPS spoofer. I used it unreasonably for the first times that I used it and i got a warning from Niatic that I may be banned. This is an automated response that will appear if you are jumping from one geographic coordinate to another in a very small amount of time. What I did was spoof to one location, log off for a day, and then travel across the country with my spoofer, log in, and it didnt suspect a thing.

**Comment by AnthonyOSX** [comment]
"Walking algorithm" will be implemented shortly.

**Comment by wseery** [comment]
@AnthonyOSX maybe you could have it teleport within a range of like a few miles to get rare pokemon?

**Comment by harsh1412** [comment]
@AnthonyOSX you can use the walking algo on PokemonGo-Bot <3

**Comment by Minnieal28** [comment]
As long as you wouldn't have the algorithm teleport, it shouldn't ban you. You could move at speeds of up to 500 mph (as if you were playing in an airplane) and the game would recognize that you were moving at a high speed, but it still wouldn't enact a ban.
You could use this to leapfrog to nearby rares.
